src/main/java/gov/nasa/pds/api/engineering/controllers/MyProductsApiController.java

```diff
@@ -61,7 +69,152 @@ public MyProductsApiController(ObjectMapper objectMapper, HttpServletRequest req
     	return this.getProductResponseEntity(lidvid);
     }
 
-    
- 
-    
+
+	@Override
+	public ResponseEntity<Products> bundlesContainingProduct(String lidvid, @Valid Integer start, @Valid Integer limit,
+			@Valid List<String> fields, @Valid List<String> sort, @Valid Boolean summaryOnly) {
+		String accept = this.request.getHeader("Accept");
+		MyProductsApiController.log.info("accept value is " + accept);
+
+		if ((accept != null 
+				&& (accept.contains("application/json") 
+						|| accept.contains("text/html")
+		 				|| accept.contains("application/xml")
+		 				|| accept.contains("application/pds4+xml")
+		 				|| accept.contains("*/*")))
+		 	|| (accept == null))
+		{
+			try
+			{
+		 		Products products = this.getContainingBundle(lidvid, start, limit, fields, sort, summaryOnly);		 		
+		 		return new ResponseEntity<Products>(products, HttpStatus.OK);
+			}
+			catch (IOException e)
+			{
+				log.error("Couldn't serialize response for content type " + accept, e);
+				return new ResponseEntity<Products>(HttpStatus.INTERNAL_SERVER_ERROR);
+			}
+		 }
+		 else return new ResponseEntity<Products>(HttpStatus.NOT_IMPLEMENTED);
+	}
+
+
+	private Products getContainingBundle(String lidvid, @Valid Integer start, @Valid Integer limit,
+			@Valid List<String> fields, @Valid List<String> sort, @Valid Boolean summaryOnly) throws IOException
+	{
+		if (!lidvid.contains("::") && !lidvid.endsWith(":")) lidvid = this.getLatestLidVidFromLid(lidvid);
+    	MyProductsApiController.log.info("find all bundles containing the collection lidvid: " + lidvid);
+    	HashSet<String> uniqueProperties = new HashSet<String>();
+    	Products products = new Products();
+    	BoolQueryBuilder query = QueryBuilders.boolQuery();
+    	SearchRequest request = new SearchRequest(this.esRegistryConnection.getRegistryIndex());
+    	SearchResponse response;
+    	SearchSourceBuilder builder = new SearchSourceBuilder();
+      	Summary summary = new Summary();
```

Comment:
Indentation

Reply:
fixed - maybe. eclipse is hiding it but git thinks the file changed so it should be fixed.